### ISSUE TITLE
Support table/column name escaped by square brackets

### ISSUE THREAD

**Issue by k-kagurazaka** [gfx/Android-Orma#326]
Hi, I'm trying to migrate DB management from directly using SQLiteDatabase to Orma.
Now, I have a DB that has table/column name escaped by square brackets like:

```java
db.execSQL("CREATE TABLE [todos] ([id] INTEGER PRIMARY KEY, [note] TEXT NOT NULL)");
db.execSQL("CREATE INDEX index_note_on_todos ON [todos] ([note])");
db.execSQL("INSERT INTO [todos] ([note]) values ('todo v1 #1'), ('todo v1 #2')");
```

I created a model class to represent "todos" table:

```java
@Table("todos")
public class Todo {
    @PrimaryKey(autoincrement = true)
    @Column(indexed = true)
    public long id;

    @Column(indexed = true)
    public String note;
}
```

However, SchemeDiffMigration is failed with the below error:
```

android.database.sqlite.SQLiteException: near ")": syntax error (code 1): , while compiling: INSERT INTO `__temp_todos` () SELECT  FROM `[todos]`
```

I also tried the model class that has square brackets escaped column name:

```java
@Table("todos")
public class Todo {
    @PrimaryKey(autoincrement = true)
    @Column(value = "[id]", indexed = true)
    public long id;

    @Column(value = "[note]", indexed = true)
    public String note;
}
```

But SchemeDiffMigration is also failed:

```
android.database.sqlite.SQLiteException: no such table: [todos] (code 1): , while compiling: INSERT INTO `__temp_todos` (`[id]`, `[note]`) SELECT `[id]`, `[note]` FROM `[todos]`
```

I assume that Orma cannot handle square brackets escaping.
Would you have a plan to support square brackets?
Thanks.

**Comment by gfx** [comment]
I see. As you said Orma can't handle square brackets. Will support them in a future.

**Comment by k-kagurazaka** [comment]
Thanks for your quick reply.
I'm looking forward to it.

**Comment by gfx** [comment]
This issue has been fixed in v3.2.1. Thanks for reporting!
